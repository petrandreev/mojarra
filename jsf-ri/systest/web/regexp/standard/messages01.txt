(?s).*<head>\s*<title>\s*messages01.jsp\s*</title>\s*<style\s*type="text/css"\s*media="screen">.*\.errors.*</style>\s*</head>\s*<body>\s*<ul\s*class="errors">\s*<li>\s*\{0\}:\s*Validation\s*Error:\s*Value\s*is\s*required.\s*</li>\s*</ul>\s*</body>.*
