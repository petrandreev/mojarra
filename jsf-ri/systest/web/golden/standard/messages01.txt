<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
  <head>
    <title>messages01.jsp</title>



    
    

<STYLE TYPE="text/css" MEDIA=screen>
<!--
.errors { 
  background-color: #7171A5;
  border: 5px outset #71A5A5;
  border-collapse: collapse;
  font-family: sans-serif;
  font-size: 14pt;
  padding: 10px;
}
-->
</STYLE>
  </head>

  <body>




<span class="errors">	Validation Error: Value is required. </span><span class="errors">	Validation Error: Specified attribute cannot be converted to proper type. </span> 





  </body>
</html>
